### ISSUE TITLE
bookmarks app not available in NC 12 beta2 ?

### ISSUE THREAD

**Issue by nursoda** [nextcloud/bookmarks#356]
The app states that it's compatible with NC 12 but I don't see it in my one already upgraded instance in 'apps'. The NC 11 instances show it (and the update to 0.10).

**Comment by marcelklehr** [comment]
My guesses are, it might not be enabled, yet, or the app catalog hasn't refreshed yet. After all, the nextcloud store shows it's compatible with Nc12: https://apps.nextcloud.com/apps/bookmarks

**Comment by blizzz** [comment]
@nursoda as @marcelklehr said it is likely that on older result was cached. Check out the apps admin page again now, or perhaps after 24hrs (not sure when the cache runs out), and report back.

**Comment by nursoda** [comment]
Well ... NC 12 upgrade and Bookmarks 0.10 release are more than 24h ago, yet no change. Plus: If it's a server? cache issue, a hint on how to resolve it would have been helpful.

Here's what I did and see:

- Logged in again, entered "apps", no change: no "Bookmarks" app. At least I did not notice it.
- Since there was NC12b3, I gave that a try. Updater appeared fine, but did not start upon clicking "Start update". Restarted browser, changed nothing. Reverted to command line, using "sudo -u www-data php updater.phar" from within /updater. Worked. But I was irritated it said I'd be using updater app version 11.0.3. Nevertheless, update was fine. Although I let it update databases on CLI, the web interface wanted to do that, too. I let it. After that, NC was back to normal except for my old ownbackup signature invalid issue. Resolved that by NC logout, "rm -r /apps/ownbackup", "redis-cli" "flushall", NC login and "rescan" from the web interface admin page. OK, so NC12b3 seems ready.
- Voila, Bookmarks is there. Installed, fine.

Lessons learned: In my opinion, the update process from 11 to 12 and within 12 betas is not robust. There are unresolved issues in app store / cache / bookmaraks, and in signature validation / cache / ownbackup.

Anyway, I think that this bug can be closed for now.
